http-server-netty/src/main/java/io/micronaut/http/server/netty/handler/accesslog/element/BytesSentElement.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.server.netty.handler.accesslog.element;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * BytesSentElement LogElement. The bytes sent.
+ *
+ * @author croudet
+ * @since 2.0
+ */
+class BytesSentElement implements LogElement {
```

Comment:
since this class is internal maybe make it final

Reply:
Done